### PR TITLE
Add CI/CD and update event schema

### PR DESCRIPTION
**Pull request by ikohobaro**
Signed-off-by: Ikoh Obaro <obaro.ikohpgs@stu.cu.edu.ng>

### CODE REVIEW

File: cmd/rocketchat/rocketchat.go [hunk 1]
```diff
@@ -693,88 +825,84 @@ func (j *DSRocketchat) GetModelData(ctx *shared.Ctx, docs []interface{}) (data *
...
 		}
 		// Mentions
 		mentionsAry, okMentions := doc["mentions"].([]map[string]interface{})
 		if okMentions {
-			mentionType := "rocketchat_message_mention"
 			for _, mentionData := range mentionsAry {
 				// map[id:XYZ name:RJ username:rjones]
 				name, _ := mentionData["name"].(string)
 				username, _ := mentionData["username"].(string)
 				name, username = shared.PostprocessNameUsername(name, username, "")
```
Comment (lukaszgryglicki) [inline cmd/rocketchat/rocketchat.go:855]: Make suere you `don't` call `name, username = shared.PostprocessNameUsername(name, username, "")` - this is to transform user data and it is supposed to be used only in `V1`.

File: swagger/rocketchat.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ definitions:
     properties:
       BackendName:
```
Comment (lukaszgryglicki) [inline swagger/rocketchat.yaml:30]: Please delete swagger file and all references to `swagger` word across this repo. Swager was replaced by `lfx-event-schema`.

Comment (ikohobaro) [reply swagger/rocketchat.yaml:30]: Sure.

File: cmd/rocketchat/rocketchat.go (hunk 1 above)
Comment (ikohobaro) [reply cmd/rocketchat/rocketchat.go:855]: Okay

File: cmd/rocketchat/rocketchat.go [hunk 3]
```diff
@@ -611,65 +726,82 @@ func (j *DSRocketchat) GetModelData(ctx *shared.Ctx, docs []interface{}) (data *
...
+					Source:     source,
+				},
+			}
+			contributors = append(contributors, contributor)
 		} else {
+			key = "created"
 			name, _ := doc["user_name"].(string)
 			// We can consider using 'user_id' if name is empty
 			username, _ := doc["user_username"].(string)
 			name, username = shared.PostprocessNameUsername(name, username, "")
```
Comment (lukaszgryglicki) [inline cmd/rocketchat/rocketchat.go:751]: this line should be commented out `name, username = shared.PostprocessNameUsername(name, username, "")` and all similar lines.

**Pull request merged**